towers/clio/level_004.rb

```diff
@@ -1,15 +1,24 @@
-#  ---
-# |@ >|
-#  ---
+#  -------
+# |    .  |
+# |@   G >|
+# |    .  |
+#  -------
 
-description "TODO"
-tip "TODO"
+description "The vacation castle is in sight, but it is guarded. That guard looks .... strong."
+tip "The guard will only let you through if you have collected the survey data the King demands"
 
 time_bonus 20 #TODO: Refine
 ace_score 19 #TODO: Refine
-size 3, 1
-stairs 2, 0
+size 7, 3
+stairs 6, 1
 
-warrior 0, 0, :east do |u|
+warrior 0, 1, :east do |u|
   u.add_abilities :walk!, :feel, :direction_of_stairs, :talk!
 end
+
+unit :trap, 4, 0
+unit :trap, 4, 2
+unit :guard, 4, 1, :west do |u|
+  u.success = "Well done, Warrior. The King will be pleased."
+  u.failure = "Pathetic. This will not do, it won't do at all."
```

Comment:
Do we tell the player explicitly how to make notes anywhere, i.e. via the `RubyWarrior::Note` model? Or do we still need to add that in?

Reply:
This still needs to be added in. It's an ability to add in the level definition.

Reply:
Should writing a note be wrapped up an ability? Or should we leave the model exposed as-is so that the player can interface with it / call ActiveRecord methods directly?

Reply:
The latter I think. That's meant to be the hardest level after all, so people will need to figure shit out.